### PR TITLE
test: remove duplicate tests

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### Motivation / Use-Case

remove duplicate tests.

### Breaking Changes
None

### Additional Info
No

### CODE REVIEW

File: test/server/Server.test.js [hunk 1]
```diff
@@ -1,107 +0,0 @@
...
-      server.startCallback((error) => {
-        if (error) {
-          throw error;
-        }
-
-        getEntries(server);
-      });
-    });
-
-    it("add hot-only option", (done) => {
```
Comment (snitin315) [inline test/server/Server.test.js:59]: e2e tests already present for `hot: true` and `hot: 'only'` - https://github.com/webpack/webpack-dev-server/blob/master/test/e2e/hot-and-live-reload.test.js

File: test/server/Server.test.js [hunk 2]
```diff
@@ -1,107 +0,0 @@
...
-        if (error) {
-          throw error;
-        }
-
-        getEntries(server);
-      });
-    });
-
-    // TODO: remove this after plugin support is published
-    it("should create and run server with old parameters order and log deprecation warning", (done) => {
```
Comment (snitin315) [inline test/server/Server.test.js:79]: E2E test already present. https://github.com/webpack/webpack-dev-server/blob/317e4b9d5c94212d2d481e7cea4ab3f40809cca6/test/e2e/api.test.js#L398

**Review by alexander-akait — APPROVED** [review]
Our CI is now more unstable, we should fix it...

**Comment by alexander-akait** [comment]
@snitin315 Feel free to merge when CI will green

**Pull request merged**